runconfig/opts/opts_test.go

```diff
@@ -61,6 +61,41 @@ func TestValidateEnv(t *testing.T) {
 	}
 }
 
+func TestValidateArg(t *testing.T) {
+	valids := map[string]string{
+		"_=a":                "_=a",
+		"var1=value1":        "var1=value1",
+		"_var1=value1":       "_var1=value1",
+		"var2=value2=value3": "var2=value2=value3",
+		"var3=abc!qwe":       "var3=abc!qwe",
+		"var_4=value 4":      "var_4=value 4",
```

Comment:
Can you add a case for a value with spaces? e.g. `--build-arg "FOO=asdf qwerty"`